src/identity-handler.js

```diff
@@ -33,11 +40,17 @@ IdentityHandler.prototype.onLoginComplete = function(
 IdentityHandler.prototype.onLogoutComplete = function(
     mParticleUser,
     identityApiRequest
-) {};
+) {
+    window.heap.resetIdentity();
+};
 IdentityHandler.prototype.onModifyComplete = function(
     mParticleUser,
     identityApiRequest
-) {};
+) {
+    if (forwarderSettings.userIdentificationType == type) {
+        heap.identify(id);
+    }
+};
```

Comment:
Two questions:

1. Where are `type` and `id` coming from? I don't see them in the function's scope.
2. Why are you calling `heap.identify` and not `window.heap.identify`? It may sound nitpicky, but I think we should be consistent.

Reply:
This was an oversight in how I combined two PRs. This code was supposed to be down in `onSetUserIdentity`, not in `onModifyComplete`. I've moved it to the proper location and also made some changes based on your feedback. Let me know how it looks.